res/css/photon/checkbox.css

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.photon-checkbox {
+  position: relative;
+  padding: 0;
+  border: 1px solid var(--grey-90-a30);
```

Comment:
there's a subtle problem with the focus style and the border, you can see it when there's a focus indicator on an unchecked checkbox: the border needs to be removed in `:focus` styles (this is actually what the photon spec says).
My preference would be to just set `.photon-checkbox:focus { border: none }` here (with a reference to the `focus.css` file).

The underlying issue is that the focus indicator uses both an inset and non-inset box shadow, and the border goes in-between, and as a result we clearly can see this gap.
(we have a similar problem for the "Docs" button at the top right, I've yet to file a bug about it)